packages/core/src/api/thread.ts

```diff
@@ -37,38 +38,52 @@ export class ThreadsAPI {
 	 * @see {@link https://discord.com/developers/docs/resources/channel#start-thread-from-message}
 	 * @see {@link https://discord.com/developers/docs/resources/channel#start-thread-without-message}
 	 * @param channelId - The id of the channel to start the thread in
-	 * @param data - The data to use when starting the thread
+	 * @param body - The data to use when starting the thread
+	 * @param options - The options to use when starting the thread
 	 */
-	public async create(channelId: Snowflake, { message_id, ...body }: StartThreadOptions) {
-		return this.rest.post(Routes.threads(channelId, message_id), { body }) as Promise<RESTPostAPIChannelThreadsResult>;
+	public async create(
+		channelId: Snowflake,
+		{ message_id, ...body }: StartThreadOptions,
```

Comment:
For a follow-up PR, this should be separated in two — `messageId: Snowflake` and `body: Omit<StartThreadOptions, 'message_id'>`.